internal/cmd/configssh.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const sshStartToken = "# ------------START-CODER-ENTERPRISE-----------"
```

Comment:
We can change these, but some of the code looks for this token, so we'll have to handle both cases (old heading and new heading) and I'm not sure if it's worth it